src/app/components/result-box/result-box.component.ts

```diff
@@ -3,8 +3,9 @@ import { Component, Input } from '@angular/core';
 @Component({
   selector: 'app-result-box',
   templateUrl: './result-box.component.html',
-  styleUrls: ['./result-box.component.scss']
+  styleUrls: ['./result-box.component.scss'],
 })
 export class ResultBoxComponent {
-  @Input() word = ''
+  @Input() word = '';
+  @Input() borderResultStyles: boolean | null = null;
 }
```

Comment:
borderStyles(): string {
     return this.borderResultStyles ? 'border-success' : 'border-danger';
}